crates/ruma-state-res/Cargo.toml

```diff
@@ -15,7 +15,11 @@ edition = "2018"
 all-features = true
 
 [features]
-unstable-pre-spec = ["ruma-events/unstable-pre-spec"]
+unstable-pre-spec = ["ruma-common/unstable-pre-spec", "ruma-events/unstable-pre-spec"]
```

Comment:
Unless state-res actually needs pre-spec things from `ruma-common`s with its own pre-spec feature, this should be left as-is.